src/engine/renderer/tr_bsp.cpp

```diff
@@ -6737,6 +6740,16 @@ void R_BuildCubeMaps()
 			tr.refdef.pixelTargetWidth = REF_CUBEMAP_SIZE;
 			tr.refdef.pixelTargetHeight = REF_CUBEMAP_SIZE;
 
+			if ( glConfig2.materialSystemAvailable ) {
+				tr.refdef.pixelTarget = nullptr;
```

Comment:
Why render it twice? Please add an explanation

Reply:
Done.